### PR TITLE
Create SliderBanner component

### PR DESCRIPTION
**Pull request by Taps73r**
Create SliderBanner component:
1) SliderBanner with react-slick and scss styles
2) Response for mobile and other devices.
3) SampleArrow component for slider.
Update package.json:
Added new dependencies:
"@types/react-slick", "react-slick", "slick-carousel" for slider working.

### CODE REVIEW

File: frontend/src/components/SliderButtons/SampleNextArrow.tsx
Comment (Kumala3) [inline frontend/src/components/SliderButtons/SampleNextArrow.tsx:1]: ### Great naming for the arrow components!
#### Change "SampleNextArrow" function name and the component and do the same with "PrevArrow" and don't forget about styles component name too!

**Comment by Taps73r** [comment]
Can you review all the changes in the new commit? All the modifications made are described there.

File: frontend/package.json
Comment (Kumala3) [inline frontend/package.json:1]: ### Update dependencies up to date with the develop branch please!

File: frontend/src/components/SliderButtons/NextArrow.tsx
Comment (Kumala3) [inline frontend/src/components/SliderButtons/NextArrow.tsx:1]: Make empty line between interface and imports as you do everywhere other.

File: frontend/src/components/SliderButtons/SliderButton.scss
Comment (Kumala3) [inline frontend/src/components/SliderButtons/SliderButton.scss:1]: Make empty lines between styles for more readable, please.

File: frontend/src/components/SliderBanner/SliderBanner.tsx
Comment (Kumala3) [inline frontend/src/components/SliderBanner/SliderBanner.tsx:1]: Update your SliderBanner up to date with develop branch. I mean change string param to Banner from types and make only one empty line between imports and interface not two.

**Comment by Taps73r** [comment]
Check new changes in PR.
Merge develop branch with feature/component/sliderbanner branch.
Resolve conflict in package.json.
Change interface in SliderBanner.
Change name for props in SliderBanner.

**Review by Kumala3 — CHANGES_REQUESTED** [review]
Delete unnecessary images from the project.
Don't forget to add a newline at the end of code, everywhere where it isn't.

**Comment by Taps73r** [comment]
Delete test pictures from the project.
Add newline at the end

**Comment by Taps73r** [comment]
### **Changes in the SliderBanner.scss file:**

- Made the height and width of the slide dependent on the device screen width.
- Reduced the spacing between slides.
- Increased the value of transform: scale.

### **Changes in the SliderBanner.tsx file:**

- Modified responsive settings by adding onFocus for mobile phones and tablets, removed unnecessary settings.
- Adjusted the interface according to the new responsive settings.

**Comment by Taps73r** [comment]
### .scss flies change:
add background for buttons,
delete border for buttons and change button size,
in all .scss files add 4 space for better reading.

### NextArrow.tsx PrevArrow.tsx change:
change div to button and fix interface

**Review by Kumala3 — APPROVED** [review]
### Great job, keep it up! PR include everything what was described in #17.
PR can be merged with the develop branch.

**Pull request merged**